packages/mobile/src/app/App.tsx

```diff
@@ -66,6 +67,23 @@ export class App extends React.Component<Props> {
   async componentDidMount() {
     await ValoraAnalytics.init()
 
+    // Handles opening Clevertap deeplinks when app is closed / in background
+    CleverTap.getInitialUrl(async (err: any, url: any) => {
+      if (err) {
+        Logger.debug('App', err)
```

Comment:
Should this be `Logger.error`? Maybe a more descriptive message? :P

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2012/commits/c5ad74ee98e6a3fa0623afd73fad0d892ed3700c!